vimsence.yaml

```diff
@@ -0,0 +1,42 @@
+filetypes:
```

Comment:
Is this file used? Why having a .json and a .yaml ?

Reply:
Yep, you're right, I'll remove it.